classes/auto-package-libs.oeclass

```diff
@@ -153,14 +153,15 @@ def auto_package_libs (d):
         files += get_extra_files(devpkg)
         d.set("FILES_" + devpkg, " ".join(files))
 
+        library_version = (d.get("LIBRARY_VERSION_" + pkg) or
+                           d.get("LIBRARY_VERSION"))
+
         pkg_provides = (d.get("PROVIDES_" + pkg) or "").split()
         pkg_provides.append("%s%s"%(provideprefix,
                                     lib.replace("_", "-").lower()))
-        library_version = (d.get("LIBRARY_VERSION_" + pkg) or
-                           d.get("LIBRARY_VERSION"))
```

Comment:
Why this code movement? Is there something I'm not seeing?

Reply:
Nope, nothing to see.  I just thought it would be a bit more clear to not hide this in the middle of the pkg_provides mangling block, when it is now also used later on.  But it is just cosmetic.